homeassistant/components/dwd_weather_warnings/config_flow.py

```diff
@@ -0,0 +1,111 @@
+"""Config flow for the dwd_weather_warnings integration."""
+
+from __future__ import annotations
+
+from typing import Any, Final
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_MONITORED_CONDITIONS, CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.selector import (
+    SelectOptionDict,
+    SelectSelector,
+    SelectSelectorConfig,
+)
+
+from .const import (
+    ADVANCE_WARNING_SENSOR,
+    CONF_OLD_REGION_NAME,
+    CONF_REGION_IDENTIFIER,
+    CURRENT_WARNING_SENSOR,
+    DEFAULT_MONITORED_CONDITIONS,
+    DEFAULT_NAME,
+    DOMAIN,
+    LOGGER,
+)
+
+CONFIG_SCHEMA: Final = vol.Schema(
+    {
+        vol.Required(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Required(CONF_REGION_IDENTIFIER): cv.string,
+        vol.Required(
```

Comment:
Why not skip this and setup both directly?

Reply:
Same here, maybe the user doesn't want to monitor both current and advance weather warnings. Presetting this would create unnecessary sensors for some users imo

Reply:
Users can always disable/enable sensors after setup so same answer as for name.
If you believe more users are for current then you can always set the advance as disabled by default (and vice versa)